src/esp/gfx/GenericDrawable.cpp

```diff
@@ -132,6 +133,16 @@ void GenericDrawable::draw(const Mn::Matrix4& transformationMatrix,
 
   updateShaderLightingParameters(transformationMatrix, camera);
 
+  // Find determinant to calculate backface culling winding dir
+  auto rotScale = transformationMatrix.rotationScaling();
+  const float normalDet = rotScale.determinant();
+  auto invNormalMat = rotScale.comatrix().transposed() / normalDet;
```

Comment:
A comment explaining the relation between a comatrix and inverse wouldn't hurt, I think. Something like this (which also includes the transpositions cancelling each other out).

```cpp
  // Normal matrix is calculated as `m.inverted().transposed()`, and
  // `m.inverted()` is the same as `m.comatrix().transposed()/m.determinant()`. 
  // We need the determinant to figure out the winding direction as well, thus 
  // we calculate it separately and then do 
  // `(m.comatrix().transposed()/determinant).transposed()`, which is the same 
  // as `m.comatrix()/determinant`.
```

Same for the PBR drawable.